apps/liquidation-bot/src/index.ts

```diff
@@ -159,11 +150,17 @@ const run = async (env: Env) => {
     timestamp: getNowSeconds(),
   });
 
-  for (let i = 0; i < riskyAccounts.length; i++) {
-    const riskyAccount = riskyAccounts[i];
+  if (riskyAccounts.length > 0) {
+    const riskyAccount = riskyAccounts[0];
 
     const possibleLiqs = await liq.getPossibleLiquidations(riskyAccount);
 
+    console.log(
```

Comment:
Similar here, do we need to keep this log line and do we get a datadog metric for this as well?